cppcache/src/CacheXmlCreation.cpp

```diff
@@ -50,12 +50,12 @@ void CacheXmlCreation::create(Cache* cache) {
 }
 
 void CacheXmlCreation::setPdxIgnoreUnreadField(bool ignore) {
-  // m_cache->m_cacheImpl->setPdxIgnoreUnreadFields(ignore);
+  // cache_->m_cacheImpl->setPdxIgnoreUnreadFields(ignore);
```

Comment:
I think you might have change this by mistake?

Reply:
Yep, CLion did it for me and I did not check it.